### ISSUE TITLE
On Network Design Spaces for Visual Recognition

### ISSUE THREAD

**Issue by sile** [sile/readings#148]
URL
-----

https://arxiv.org/abs/1905.13214

Abstract
----------

> Over the past several years progress in designing better neural network architectures for visual recognition has been substantial. To help sustain this rate of progress, in this work we propose to reexamine the methodology for comparing network architectures. In particular, we introduce a new comparison paradigm of distribution estimates, in which network design spaces are compared by applying statistical techniques to populations of sampled models, while controlling for confounding factors like network complexity. Compared to current methodologies of comparing point and curve estimates of model families, distribution estimates paint a more complete picture of the entire design landscape. As a case study, we examine design spaces used in neural architecture search (NAS). We find significant statistical differences between recent NAS design space variants that have been largely overlooked. Furthermore, our analysis reveals that the design spaces for standard model families like ResNeXt can be comparable to the more complex ones used in recent NAS work. We hope these insights into distribution analysis will enable more robust progress toward discovering better networks for visual recognition.

Table of Contents
--------------------

- (1) Introduction
- (2) Related Work
  - Reproducible research
  - Empirical studies
  - Hyperparameter search
  - Neural architecture search
  - Complexity measures
- (3) Design Spaces
  - (3-1) Definitions
    - (3-1-1) Model family
    - (3-1-2) Design space
    - (3-1-3) Model distribution
    - (3-1-4) Data generation
  - (3-2) Instantiations
- (4) Proposed Methodology
  - (4-1) Comparing Distributions
    - Point estimates
    - Distributions
    - Discussion
  - (4-2) Controlling for Complexity
    - Unnormalized comparison
    - Error vs complexity
    - Complexity distribution
    - Normalized comparison
  - (4-3) Characterizing Distributions
    - Distribution shape
    - Distribution area
    - Random search efficiency
  - (4-4) Mnimal Sample Size
    - Qualitative analysis
    - Quantitative analysis
    - Feasibility discussion
- (5) Case Study: NAS
  - (5-1) Design Spaces
  - (5-2) Design Space Comparisons
  - (5-3) Comparisons to Standard Design Spaces
  - (5-4) Sanity Check: Point Comparisons
- (6) Conclusion
- (A) Supporting Experiments
  - Training schedule
  - Training settings
  - Model consistency
  - Bucket comparison
- (B) ImageNet Experiments

BibTeX
------

```bibtex
@misc{radosavovic2019network,
    title={On Network Design Spaces for Visual Recognition},
    author={Ilija Radosavovic and Justin Johnson and Saining Xie and Wan-Yen Lo and Piotr Dollár},
    year={2019},
    eprint={1905.13214},
    archivePrefix={arXiv},
    primaryClass={cs.CV}
}
```
